src/Avalonia.Themes.Default/MenuItem.xaml

```diff
@@ -36,12 +47,15 @@
                 </DataTemplate>
               </ContentPresenter.DataTemplates>
             </ContentPresenter>
+            <TextBlock Grid.Column="3"
```

Comment:
I think having a name assigned here will make styling easier. In our case we would like this text to have different color.

Reply:
Done!